rules/opentitan_test.bzl

```diff
@@ -25,6 +25,7 @@ _BASE_PARAMS = {
     "rom": "@//sw/device/lib/testing/test_rom",
     "tags": [],
     "test_runner": "@//util:opentitan_functest_runner.sh",
+    "test_script": [],
```

Comment:
Let me know if there is a better name for the required args / commands and options that run the test. I'm not sure this is the most helpful one for readability, but good ideas didn't manifest, haha.

Reply:
`test_cmds` ?

Reply:
Done! Thanks for helping out!